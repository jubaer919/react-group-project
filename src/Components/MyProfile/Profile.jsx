export default function Profile() {
  return (
    <>
      <div>
        <h1>Mission </h1>
      </div>
    </>
  );
}
