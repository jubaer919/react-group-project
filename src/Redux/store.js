import { configureStore } from '@reduxjs/toolkit';
import rocketReducer from './Rocket/rocketSlice';
import missionReducer from './missions/missions';

const store = configureStore({
  reducer: {
    rocket: rocketReducer,
    missions: missionReducer,
  },
});

export default store;
