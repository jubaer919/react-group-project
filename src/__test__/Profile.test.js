import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Profile from '../Components/MyProfile/Profile';
import store from '../Redux/store';

describe('Test for Rockets', () => {
  test('should render', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Profile />
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
