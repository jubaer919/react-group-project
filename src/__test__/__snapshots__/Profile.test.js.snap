// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Rockets should render 1`] = `
<div
  className="profile-container"
>
  <div
    className="mission-container"
  >
    <h2
      className=""
    >
      My Missions
    </h2>
    <ul>
      <li
        className=""
      >
        None
      </li>
    </ul>
  </div>
  <div
    className="rocket-container"
  >
    <h2>
      Rocket
    </h2>
    <ul />
  </div>
</div>
`;
